packages/bootleg-app-contracts/migrations/2_deploy_and_mint.js

```diff
@@ -1,17 +1,28 @@
 const BootlegToken = artifacts.require('BootlegToken');
+const BootlegTraderApp = artifacts.require('BootlegTraderApp');
 
-module.exports = async deployer => {
-  const tokenId = web3.utils.toBN(1);
+module.exports = async (deployer, network, accounts) => {
+  let firstOwner = web3.utils.toHex(0);
```

Comment:
Checking, this should probably be the band address in real life?

Reply:
Yes, would be the bands address.

Reply:
Hi All, we haven't been able to get in the loop on Bootleg. Just want to throw it out there that the band is often NOT the rights holder for the livestreaming, broadcast, or VoD.

Reply:
That's a good point. @breakpointer This migration script is very specific to us. 

We should leave some comments to that point that this is really an example migration for our specific use case. Folks who make it this far will need to take this one as a base and change it to their needs. 

For @mgw-sbic's use case, I believe you would actually want to deploy the token contract and not mint anything in the migration but instead have a server running that, at the time of the livestream, mint a new token with the correct array positions.

Reply:
@breakpointer Should we maybe have the `firstOwner` an env variable so we can pass it in? 

```suggestion
  let firstOwner = process.env.BAND_ADDRESS || web3.utils.toHex(0);
```